pytorch_lightning/core/memory.py

```diff
@@ -259,11 +286,11 @@ def __str__(self):
         if self._model.example_input_array is not None:
             arrays.append(["In sizes", self.in_sizes])
             arrays.append(["Out sizes", self.out_sizes])
+        total_parameters = self.total_parameters
+        trainable_parameters = self.trainable_parameters
+        model_size = self.model_size()
```

Comment:
I'd suggest either make the above 2 as methods or make model_size a property too. WDYT?

Reply:
model_size should be property, I agree was about to change it, thanks for pointing it out :)